common/src/main/java/org/apache/syncope/common/services/TaskService.java

```diff
@@ -242,4 +244,24 @@ TaskExecTO execute(@NotNull @PathParam("taskId") Long taskId,
     @Produces({ MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON })
     @Consumes({ MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON })
     BulkActionResult bulk(@NotNull BulkAction bulkAction);
+
+    /**
+     * List all the tasks of a particular type
```

Comment:
Please change this description: it should be something like as "List task jobs of the given type"